www/src/components/shell/constants.js

```diff
@@ -1,3 +1,10 @@
+// START <<Remove this after dev>>
+export const DEBUG_SCM_TOKENS = {
+  GITLAB: '',
+  GITHUB: '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',
```

Comment:
these do expire so not a huge deal, but should probably still rm

Reply:
Done.